uqcsbot/scripts/welcome.py

```diff
@@ -10,9 +10,16 @@
     "Hey there! Welcome to the UQCS slack!",
     "This is the first time I've seen you, so you're probably new here",
     "I'm UQCSbot, your friendly (open source) robot helper",
-    "We've got a bunch of generic channels (e.g. #banter, #games, #projects) along with many " \
+    "We've got a bunch of generic channels (e.g. #banter, #memes, #projects) along with many " \
```

Comment:
would prefer to keep games and choose between either banter or memes for diversity sake 👍